system/modules/search/models/SearchService.php

```diff
@@ -93,7 +93,7 @@ public function getResults($query, $index = null, $page = null, $pageSize = null
 		$index_all_limit = 10;
 		
 		$select = "SELECT class_name, object_id ";
-        $from = " FROM object_index WHERE object_id != 0 AND MATCH (content) AGAINST ('$str' IN $index_mode) ";
+        $from = " FROM object_index WHERE object_id != 0 AND MATCH (content) AGAINST (".$this->_db->quote($str)." IN $index_mode) ";

```

Comment:
I had to quote the string here as tags such as "Won't fix" would throw an error.